input_parser.star

```diff
@@ -134,12 +137,22 @@ DEFAULT_L1_ARGS = {
     # "minimal" preset will spin up a network with minimal preset. This is useful for rapid testing and development.
     # 192 seconds to get to finalized epoch vs 1536 seconds with mainnet defaults
     # Please note that minimal preset requires alternative client images.
-    "l1_preset": "minimal",
+    "l1_preset": "mainnet",
     # Number of seconds per slot on the Beacon chain
     # Default: 12
     "l1_seconds_per_slot": 1,
-    # The amount of ETH sent to the admin, sequence, aggregator and sequencer addresses.
-    "l1_funding_amount": "100ether",
+    # The amount of ETH sent to the admin, sequence, aggregator, sequencer and other chosen addresses.
+    "l1_funding_amount": "1000000ether",
+    # Default: 2
+    "l1_participants_count": 1,
+    # Default: 2048
+    "l1_eth1_follow_distance": 1,
+    # Default: 256
+    "l1_min_validator_withdrawability_delay": 1,
+    # Default: 256
+    "l1_shard_committee_period": 1,
```

Comment:
Leaving more context in comments:
[SHARD_COMMITTEE_PERIOD](https://eth2book.info/capella/part3/config/configuration/#shard_committee_period)
This really anticipates the implementation of data shards, which is no longer planned, at least in its originally envisaged form. The [idea is](https://github.com/ethereum/consensus-specs/issues/675#issuecomment-468159678) that it's bad for the stability of longer-lived committees if validators can appear and disappear very rapidly. Therefore, a validator cannot initiate a voluntary exit until SHARD_COMMITTEE_PERIOD epochs after it has been activated. However, it could still be ejected by slashing before this time.

Reply:
Same here